src/Views/Form/Templates/Sequoia/optionConfig.php

```diff
@@ -6,7 +6,7 @@
 $formInfo = get_post( FrontendFormTemplateUtils::getFormId() );
 
 // Setup dynamic defaults
-$introHeadline    = $formInfo->post_title ? $formInfo->post_title : __( 'Campaign Heading', 'give' );
+$introHeadline    = ($formInfo->post_title && 'Auto Draft' !== $formInfo->post_title ) ? $formInfo->post_title : __( 'Support Our Cause', 'give' );
```

Comment:
@DevinWalker This logic will break for non-english sites.

![image](https://user-images.githubusercontent.com/1784821/84934763-6834fa80-b0f5-11ea-9757-7be353f36ce4.png)

Please check if admin creating a new form or not instead of comparing the title.